libraries/AP_HAL_ChibiOS/hwdef/NucleoH743-IKS02/hwdef.dat

```diff
@@ -0,0 +1,129 @@
+# hw definition file for processing by chibios_hwdef.py
+# for H743 bootloader
+
+AUTOBUILD_TARGETS None
+
+# MCU class and specific type
+MCU STM32H7xx STM32H743xx
+
+# crystal frequency
+OSCILLATOR_HZ 8000000
+
+define STM32_HSE_BYPASS
+
+# board ID for firmware load
+APJ_BOARD_ID 139
```

Comment:
This probably deserves a separate board ID; this new firmware is not going to boot on a board without the IMU available... you can reserve another id in board_types.txt (above 1000)